tools/python/maps_generator/README.md

```diff
@@ -42,10 +42,12 @@ maps_generator$ vim var/etc/map_generator.ini
 
 ```ini
 [Main]
-# The path where the planet will be downloaded and the maps are generated.
-MAIN_OUT_PATH: ~/maps_build
 # If the flag DEBUG is set a special small planet file will be downloaded.
 DEBUG: 1
+# The path where the planet will be downloaded and the maps are generated.
+MAIN_OUT_PATH: ~/maps_build
+# The path where stores caches for nodes, ways, relations.
```

Comment:
The path where caches for nodes, ... are stored

Reply:
++